Makefile.am

```diff
@@ -115,7 +115,11 @@ CLEANFILES+=\
 	$(top_builddir)/usbguard.service
 
 usbguard.service: $(top_srcdir)/usbguard.service.in
-	$(SED) -e "s|%sbindir%|${sbindir}|" -e "s|%sysconfdir%|${sysconfdir}|" $^ > $@ || rm -f $@
+	$(SED) \
+	  -e "s|%sbindir%|${sbindir}|" \
+	  -e "s|%sysconfdir%|${sysconfdir}|" \
+	  -e 's|%localstatedir%|${localstatedir}|" \
```

Comment:
Could you replace the first ```'``` wtih ```"``` otherwise it is not valid.

Reply:
Oh, sure. My bad.